bin/migrate-oats-data/noi/notice_of_intent_submissions/notice_of_intent_soil_fields.py

```diff
@@ -149,7 +149,7 @@ def _prepare_oats_alr_applications_data(row_data_list):
 
 
 def map_soil_fields(data):
-    soil_type = data.get("material_description", "")
+    soil_type = data.get("material_desc", "")
```

Comment:
This seems like a duplicate of the function from [.../noi_decisions/noi_components/notice_of_intent_decision_components_soil_details_update.py](https://github.com/bcgov/alcs/pull/1249/files#diff-0f292c4c25c2b8b0ca95bee3d7efa19b6f6d5faafb57a269846275285bebd3ca)

Reply:
correct, since the application and decision components in OATS are the same but different in ALCS. This function is intentionally duplicated since they are responsible for different ALCS tables. The only piece that is currently shared is Enum since it has constraints on DB level and it is safe to reuse that code

Reply:
Should basically be a carbon copy but with different join and insert